static/src/javascripts/types/global.d.ts

```diff
@@ -444,6 +444,7 @@ interface Window {
 			dfpEnv?: DfpEnv;
 		};
 		notificationEventHistory?: HeaderNotification[][];
+		identityAuth?: unknown;
```

Comment:
For some reason frontend's `global.d.ts` gets extremely unhappy if you try to import types otherwise I would make this not be `unknown` and use a more explicit type.